parlai/crowdsourcing/tasks/turn_annotations_static/turn_annotations_blueprint.py

```diff
@@ -78,12 +78,16 @@ class TurnAnnotationsStaticBlueprintArgs(StaticReactBlueprintArgs):
             "help": "Path to data and answers for onboarding task in JSON format"
         },
     )
-    annotation_buckets: str = field(
-        default=os.path.join(get_task_path(), 'task_config/annotation_buckets.json'),
+    annotation_buckets: Optional[str] = field(
+        default=None,
```

Comment:
I think this is a good change - it's good to have no checkboxes by default. In fact, I just made that change to the `model_chat` task as well :)